docs/install/index.md

```diff
@@ -1090,7 +1090,8 @@ You can [build MXNet-R from source](windows_setup.html#install-mxnet-package-for
   options(repos = cran)
   install.packages("mxnet")
 ```
-Change cu92 to cu80, cu90 or cu91 based on your CUDA toolkit version. Currently, MXNet supports these versions of CUDA.
+Change cu92 to cu80, cu90, cu91 or cuda 100 based on your CUDA toolkit version. Currently, MXNet supports these versions of CUDA.
```

Comment:
Shoud be `cu100` without the space. It will be part of a path.

Reply:
Aah. My bad. 
Done.